File: Java/aula1/src/BookModel.java
public class BookModel {

    // attributes - properties
    String title;
    String isbm;
    String author;
    String genre;
    int pages;
    boolean borrowed;

    // opereations - methods
    // what is returned when printing the object
    public String toString(){
        return title + ", " + author + ", " + borrowed;
    }

    // make this attributes required when creating a new object
    public BookModel(String title, String isbm, String author, String genre, int pages) {
        this.title = title;
        this.isbm = isbm;
        this.author = author;
        this.genre = genre;
        this.pages = pages;
        this.borrowed = false;
    }

}

File: C#/aula1/SistemaBiblioteca/BookModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LibrarySystem
{
    internal class BookModel
    {
        // Attributes - Properties
        public string title;
        public string isbm;
        public string author;
        public string genre;
        public int pages;
        public bool borrowed;


        // Operations - Methods
        // Make this attributes required when creating a new book
        public BookModel(string title, string isbm, string author, string genre, int pages)
        {
            this.title = title;
            this.isbm = isbm;
            this.author = author;
            this.genre = genre;
            this.pages = pages;
            this.borrowed = false; // Default value for borrowed
        }
    }
}
